config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```

Comment:
1. Config this file into .gitignore then run bellow command line --> do not push this file to pull request
`git rm --cached config/database.yml `
2. Create a new file with name "config/database.yml.example" --> this file using push to pull request

Reply:
can you help me how to resolve this problem? I do not where to start to solve this problem.

Reply:
I run this command and it appear like this
$ git add config/database.yml.example
fatal: pathspec 'config/database.yml.example' did not match any files

Reply:
Steps:
- You need to create a new file into `config/` folder which named `database.yml.example` with the same format database.yml file
- gitignore `config/database.yml` file